libyang/data.py

```diff
@@ -256,6 +256,19 @@ def meta_free(self, name):
                 lib.lyd_free_meta_single(item)
                 break
 
+    def new_meta(self, name: str, value: str, clear_dflt: bool = False):
+        ret = lib.lyd_new_meta(
+            self.context.cdata,
```

Comment:
Since you are providing a parent node, it is OK (and better in my opinion) to avoid passing a context pointer. You can replace that argument with `ffi.NULL`.